docs/extras/modules/data_connection/vectorstores/integrations/xata.mdx

```diff
@@ -0,0 +1,52 @@
+# Xata
+
+[Xata](https://xata.io) is a serverless data platform, based on PostgreSQL. It provides a type-safe TypeScript/JavaScript SDK for interacting with your database, and a UI for managing your data.
+
+Xata has a native vector type, which can be added to any table, and supports similarity search. LangChain inserts vectors directly to Xata, and queries Weaviate for the nearest neighbors of a given vector, so that you can use all the LangChain Embeddings integrations with Xata.
```

Comment:
`and queries Weaviate for the nearest neighbors of a given vector` is this a typo?

Reply:
fixed, thanks :)